route-monitor-operator/scaffold/kas-monitoring-hack.managedclustersetbinding.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: cluster.open-cluster-management.io/v1beta2
+kind: ManagedClusterSetBinding
+metadata:
+  name: default
+  namespace: openshift-route-monitor-operator
+spec:
+  clusterSet: default
```

Comment:
i already completely forgot how managedclustersets and their bindings work.
i just know we have another clusterset hypershift-management-clusters

you seem to know your way around this. can you elaborate this a bit?

Reply:
ACM uses this mechanism to impose restrictions on which users can manage which clusters using policies. In practice, when you bind a clusterset to a namespace, any user with permission to deploy policies in that namespace can apply policies to any cluster that belongs to that clusterset.

In this particular case, we are binding the "default" clusterset, which only has one member (the "local-cluster"), to the "openshift-route-monitor-operator" namespace. Effectively, any user with permissions to create/modify/delete policies in this namespace can affect the management cluster.

Reply:
ah i see. `default`== `local-cluster` `hypershift-management-clusters` == `HCPs`